static/js/components/LearnerSearch.js

```diff
@@ -45,6 +47,14 @@ const makeSearchkitTranslations: () => Object = () => {
   return translations;
 };
 
+const makeDegreeTranslations: () => Object = () => {
```

Comment:
can you change the name of `makeSearchkitTranslations` to `makeCountryNameTranslations`, and the `searchkitTranslations` variable name to `countryNameTranslations`? `makeSearchkitTranslations` was only a relevant name because it was the only translation we were applying to any of our searchkit results. now that we are creating another set of translations, we should get more specific with the naming